### PR TITLE
Derek etman

### PR DESCRIPTION
**Pull request by DerekEtman**
Here's to hoping.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -12,62 +12,96 @@
...
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+    <div class="text-container about-text">
+      <img src="img/about-plan.png" alt="strategy">
+      
+      <h2> Strategy</h2>
+        
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    Learn More
+        <a href="#">Learn More</a>
```
Comment (gisturiz) [inline about.html:51]: Unless you want something to link something on your site, it's not necessary to add an anchor tag, whose purpose is to provide a link. In this case, the Learn More buttons would have benefitted more from a <button> tag.

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                    <a href="#">Contract</a>
+                    </nav>
+                        
+        </header>
+
+    <img id="contact-image" src="img/Contact.png">
+    
+        <div class="contact-container">
+
+                <form action="/action_page.php">
```
Comment (gisturiz) [inline contact.html:37]: Awesome job researching and building out this contact form! We will be going over this later in the track, so you're a step ahead.

File: css/index.css [hunk 3]
```diff
@@ -110,8 +185,73 @@ p {
     display: flex;
     align-items: center;
     justify-content: center;
+    animation: vibrate-1 0.3s linear infinite both;
+}
+
+@keyframes vibrate-1 {
```
Comment (gisturiz) [inline css/index.css:191]: Very cool

File: css/index.css [hunk 4]
```diff
@@ -110,8 +185,73 @@ p {
...
+    80% {
+      transform: translate(2px, -2px);
+    }
+    100% {
+      transform: translate(0);
+    }
+  }
+
+/*Omg so many nth-child boxes repeating myself too much BUT OH WELL ON TO THE NEXT PROBLEM*/
+
```
Comment (gisturiz) [inline css/index.css:213]: Great job figuring out the nth-child solution.

**Review by gisturiz — APPROVED** [review]
Great job on MVP and stretch, the contact page was impressive. 3 stars

File: css/index.css (hunk 4 above)
Comment (DerekEtman) [reply css/index.css:213]: Oh Great! I thought I had done it completely wrong.

File: contact.html (hunk 2 above)
Comment (DerekEtman) [reply contact.html:37]: Thanks, I spent a long time researching it for the UI project!

**Pull request merged**